src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -4,6 +4,12 @@
 @import "~leaflet/dist/leaflet.css";
 @import "~react-tabs/style/react-tabs";
 
+// TODO: These colors are not close enough to anything that already exists in
+// Formation. Perhaps the design should be reevaluated to see if Formation
+// colors could be used in place of these
```

Comment:
Probably worth writing a ticket for this so we can track it.

Reply:
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/1689